### ISSUE TITLE
tmp/cache permissions start wrong

### ISSUE THREAD

**Issue by dreamingmind** [OrigamiStructures/sampleblog2#7]

**Comment by dreamingmind** [comment]
This may never work... or can composer resolve this? I think baking a new app can make these work but baking ain't cloning.

**Comment by curlyjason** [comment]
Yeah, composer may be able to resolve this. I need to do a little research and write my install / update article based upon composer, instead of git. I think (at this point) that these two approaches are for different goals. Using Git and Git Submodules is meant for a number of pieces in flux, with development continuing on all fronts. Composer is for a snapshot installation, with no updates or ongoing development planned.

**Comment by dreamingmind** [comment]
That makes sense. 

At any rate, I'd never really thought of doing what you have here - having a whole cake shell application with a set of 'normal' submodules. As an idea it seems very appealing. But maybe in practice, it's wouldn't be such a big deal. After all, how often do you need to stand up a new app?
